Test/TestDeconvolution.cs

```diff
@@ -15,6 +17,84 @@ namespace Test
     [ExcludeFromCodeCoverage]
     public sealed class TestDeconvolution
     {
+
+        [Test]
+        public static void CheckGetMostAbundantObservedIsotopicMass()
+        {
+            
+            string fullFilePathWithExtension = @"D:\TDBU\Jurkat\TD-Projects-JurkatTopDownSeanDaiPaper\FXN11_tr1_032017.raw"; //@"D:\Other\averageMass\for_shortreed_protein_mix_sid 15_topdown_standard-qb.raw";
```

Comment:
can't hard code files paths. have to load unit test files into mzlib for storage. make sure the add "copy always" to the properties